plugin.video.videolina/LICENSE.txt

```diff
@@ -1,339 +1,339 @@
-GNU GENERAL PUBLIC LICENSE

```

Comment:
this file shouldn't be changed in this PR

Reply:
Sorry seems it happend, by checkout to Linux... Since i cant  remove it , i make a new PR